libs/barista-components/sunburst-chart/src/sunburst-chart.scss

```diff
@@ -15,25 +21,25 @@ text {
   }
 }
 
-.dt-sunburst-slice--current {
+.dt-sunburst-chart-slice--current {
   cursor: default;
 }
 
-.dt-sunburst-label {
+.dt-sunburst-chart-label {
   transform: translateY(0.9rem);
   font-size: 0.9rem;
 }
 
-.dt-sunburst-value {
+.dt-sunburst-chart-value {
   transform: translateY(-0.1rem);
   font-size: 1rem;
 }
 
-/** Sunburst overlay */
+/** SunburstChart overlay */
 /** The selector is combined with the cdk-overlay-pane to
 increase the specificity to avoid edge cases where the cdk styles
 are loaded after the component's styles in the angular.json */
-::ng-deep .cdk-overlay-pane.dt-sunburst-overlay-panel {
+::ng-deep .cdk-overlay-pane.dt-sunburst-chart-overlay-panel {
   @include dt-overlay-container();
 
   pointer-events: none;
```

Comment:
Could you please explain why we need to set pointer-events: none here?

Reply:
no longer needed. I'm removing it